readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://YevhenRDoIT.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://YevhenRDoIT.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="1008" alt="image" src="https://github.com/user-attachments/assets/c9fd7be4-06ff-43cf-bdd5-384c1389f8ee">